MapboxNavigationTests/LaneTests.swift

```diff
@@ -1,6 +1,6 @@
 import XCTest
 import FBSnapshotTestCase
-import MapboxDirections
+@testable import MapboxDirections
```

Comment:
Getting the error: `Module 'MapboxDirections' was not compiled for testing`

Does something need to change in the cartfile to make this happen?

Reply:
Is this `@testable` needed? This PR doesn’t seem to introduce any calls to private methods.

Reply:
Yes, this init is private: https://github.com/mapbox/mapbox-navigation-ios/pull/1270/files#diff-ec73732e527cb329ec167966c37b7cbeR50

Reply:
Not sure what the best practice is here but testable imports publicizes private methods in frameworks built with its configuration set to debug (carthage update/bootstrap `--configuration Debug`)